src/modules/user/user.controller.ts

```diff
@@ -38,11 +38,11 @@ export class UserController {
 
   @Patch('/games')
   updateUserGames(
-    @ReqUser() user: User,
```

Comment:
Burda niye Requser kullanmayip body'den aliyoruz userId'yi?

Reply:
Normalde bu route u olusturdugumda game master direk kendi oyunlarına ulassın die yapmıştım.ondan req user demiştim . Ama manager users sayfasından ulaşacağı için her user için userId yollamasını istedim

Reply:
manager user'in game'lerini degistiremiyor sadece gorebiliyor. o yuzden patch'te manager erisimi olmasina gerek yok sanki?